docs/README-Developer.md

```diff
@@ -217,4 +217,15 @@ For SDK packages like `zowe-explorer-api`, a TGZ is produced that can be install
 
 For VS Code extensions, a VSIX is produced that can be installed in VS Code or published to the VS Code Marketplace or Open VSX Registry.
 
-**Note:** The list of files included in the VSIX bundle is defined by an allowlist in the ".vscodeignore" file.
+> **Note**
+> The list of files included in the VSIX bundle is defined by an allowlist in the ".vscodeignore" file.
+
+The CI workflows for Zowe Explorer and Zowe Explorer FTP run the `yarn package` command and archive VSIXs as artifacts that can be downloaded for testing. If you have the GitHub CLI installed, you can define a Bash alias like the following to download the VSIX for a given pull request (for example, `dlVsix 123`):
+
+```shell
+dlVsix() {
+  branch=$(gh pr view $1 --json headRefName --jq ".headRefName")
+  runId=$(gh run list -b $branch --limit 1 --status success --workflow "Zowe Explorer CI" --json databaseId --jq ".[0].databaseId")
+  gh run download $runId -n zowe-explorer-vsix
```

Comment:
I think there may be a way to make the alias work from any directory, going to look into it 🙂

Unfortunately I don't think there is a way to give it a custom name using the `gh` cli, and the `gh` cli doesn't print the name of the file it downloaded so it's not easy to run a `mv` command to rename the file either 😢

Reply:
the other option is to do an `ls` before and after the download command and get the diff (which should be the new vsix) and then do the `mv` 😋 

(not suggesting to do that though.
The alias works just fine as it is 🥳 

And thanks for looking into it 🥳

Reply:
the other option is to do an `ls` before and after the download command and get the diff (which should be the new vsix) and then do the `mv` 😋 

(not suggesting to do that though.
The alias works just fine as it is 🥳 

And thanks for looking into it 🥳

Reply:
Updated the alias in https://github.com/zowe/vscode-extension-for-zowe/pull/2237/commits/3020ef65938770f3f147a020dbd98b572afb2066 so it can now be run from any directory, and branch names are also supported 😋